pkgs/top-level/python-packages.nix

```diff
@@ -3195,6 +3197,9 @@ in {
 
   priority = callPackage ../development/python-modules/priority { };
 
+  privacyidea = callPackage ../servers/privacyidea { };
```

Comment:
Please don't call packages that are outside of `python-modules`. Are the packages meant as applications or not?

Reply:
They are meant to be started by uwsgi/twisted, see module, do you define that as application or not?

Reply:
Considering it needs to be imported
```nix
penv = python.withPackages (ps: [ ps.privacyidea ]);
```
I prefer to have the expression in `python-modules` but its this grey area I suppose.

Reply:
I'll move it